### DISCUSSION TITLE
How do I use rook-ceph outside the k8s cluster

### DISCUSSION THREAD

**Discussion by zhangdeshuai1999** [rook/rook#13677] [General]
Does the latest version 1.13 of rook-ceph provide storage classes for use by applications outside the k8s cluster, and what storage protocols are supported?

**Comment by travisn** [comment]
@zhangdeshuai1999 You can configure [NFS](https://rook.io/docs/rook/latest/Storage-Configuration/NFS/nfs/) or an external [S3 enpdoint](https://rook.io/docs/rook/latest/Storage-Configuration/Object-Storage-RGW/object-storage/#access-external-to-the-cluster).
